util/exec.go

```diff
@@ -23,6 +23,14 @@ type Command struct {
 // CommandWithResult executes a command, and prints and returns STDERR/STDOUT.
 func CommandWithResult(log *log.Logger, command *Command) (stdout string, stderr string, err error) {
 	log.Printf("$ %s", strings.Join(command.Command, " "))
+	defer func() {
```

Comment:
Please revert this defer and use the "log" variable instead of os.Stdout and os.Stderr

Reply:
If we use the logger instead of stdout and stderr, this clashes with https://github.com/symflower/eval-dev-quality/pull/53#discussion_r1572350887.

Reply:
I guess I will use the log in addition.

Reply:
If I change the defer approach to: 

```go
c.Stdout = io.MultiWriter(os.Stdout, &stdoutWriter, log.Writer())
c.Stderr = io.MultiWriter(os.Stderr, &stderrWriter, log.Writer())
```

I get a data race. :crying_cat_face: seems like the log writer is not thread safe?

Reply:
Also when removing the `os.Stdout` and `os.Stderr`.